chaco/plot.py

```diff
@@ -173,8 +173,8 @@ def __init__(self, data=None, **kwtraits):
             elif type(data) in (ndarray, tuple, list):
                 self.data = ArrayPlotData(data)
             else:
-                raise ValueError, "Don't know how to create PlotData for data" \
-                                  "of type " + str(type(data))
+                raise ValueError("Don't know how to create PlotData for data"
+                                 "of type {}".format(type(data)))
```

Comment:
While you're here, you could also fix the missing space between "data" and "of".